chunks/lang-go/chunk.yaml

```diff
@@ -1,7 +1,7 @@
 variants:
-  - name: "1.19.5"
+  - name: "1.19.7"
     args:
-      GO_VERSION: 1.19.5
-  - name: "1.20"
+      GO_VERSION: 1.19.7
+  - name: "1.20.2"
     args:
-      GO_VERSION: 1.20
+      GO_VERSION: 1.20.2
```

Comment:
Recent stable version is `1.20.3` now. ([reference](https://go.dev/dl/#:~:text=e12384311403f1389d14cc1c1295bfb4e0dd5ab919403b80da429f671a223507-,go1.20.3.linux%2Damd64.tar.gz,-Archive))

Reply:
1.20.4 now